.github/workflows/build_docs.yml

```diff
@@ -6,37 +6,33 @@
      - main
```

Comment:
Let's replace this with [the following](https://github.com/pysal/spaghetti/blob/42c9fb3184e9894daf80a54a1cb1ff1ea117cd90/.github/workflows/build_docs.yml#L3-L13), which either a) only rebuilds the docs when a version is released, or b) triggered manually (see example in [`spaghetti`](https://github.com/pysal/spaghetti/actions/workflows/build_docs.yml):

```
 on:
   push:
     # Sequence of patterns matched against refs/tags
     tags:
       - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
   workflow_dispatch:
     inputs:
       version:
         description: Manual Doc Build Reason
         default: test
         required: false
```